README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
+| --format_det     | Takes in keyword that represents the format that the detection will be in (either text files or COCO Detection JSON format)                           | python cli.py --format_det coco                               |         | Available inputs: `coco`, `xyrb` (x,y,x2,y2), or `xywh` (x,y,width,height)                                             |
+| --metric       | Takes in keyword that represents the metric that will be used                                                                                         | python cli.py --metric voc2012                              |         |Available options: `coco`, `voc2007`, `voc2012`, `auc`                                                                                                                        |
+| --names, -n     | The path of the file that contains the class names for this detection                                                                                 | python cli.py --names voc_classes.txt                        |         | The file with names of classes should just be a plaintext file with each class name delimited by a newline.                             |
+| --threshold, -t | Represents the IoU threshold for a given metric                                                                                                       | python cli.py --metrics voc2007 --threshold 0.75             | 0.5     |                                                                                                                        |
+| --plot, -p  | A flag for creating a Precision x Recall graph for the given metric that it is being used in                                                          | python cli.py --metrics voc2007 --plot                          |         | This flag requires the `savepath` argument to be specified                                                             |
+| --save_path, -sp | An optional parameter used to specify where the visual output will be saved to. Required when `--plot` is specified                                      | python cli.py --save_path toyexample/image_out/               |         |                                                                                                                        |
+| --coord_det      | Argument is associated with `format_det` and represents the coordinate system of the detection files. Either absolute (normal coordinates) or relative | python cli.py --format_det xyrb --coord_det rel                |         | Available inputs: `abs`, `rel`                                                                                         |
```

Comment:
The formats for detection will consider <class_id> or <class_name>?
As the parameter `--name` points to the list of names, I suppose only <class_id> are supported, right? We should support the same options as the interface.
What if the user has detections using <class_names>, as in:
`dog 0.78 13 56 100 200`? Is he able to use the CLI?

Reply:
I believe <class_name> is supported in the CLI since it passes `test_case_1`. When specified, `--names` is used for YOLO ground truth (if that gt format is specified) and to replace <class_id> with <class_name>. If `--names` is not specified, CLI will throw error if ground truth is YOLO and assume detection files are in <class_name> format. Unless I'm mistaken, I believe this follows the same behavior as the interface? Are you wanting maybe two different name arguments; one for gt and one for det?

For now, I've updated the old replacement code with the original `replace_id_with_classes` function to reduce clutter (6e1c5b129e50d895a16cd36c321624473ca02940)

I also made an additional `test_case_2`, where the detection files have <class_id> and ground truth files have <class_name> to test if the behavior when `--names` is specified is correct. I won't push the test stuff unless you are interested.